Firmware/mmu2.cpp

```diff
@@ -633,13 +637,13 @@ void MMU2::ReportError(ErrorCode ec) {
 
 void MMU2::ReportProgress(ProgressCode pc) {
     ReportProgressHook((CommandInProgress)logic.CommandInProgress(), (uint16_t)pc);
-    
+
     // Log progress - example: MMU2:P=123 EngageIdler
     char msg[64];
-    snprintf(msg, sizeof(msg), "MMU2:P=%hu", (uint16_t)pc);
+    int len = snprintf(msg, sizeof(msg), "MMU2:P=%hu ", (uint16_t)pc);
```

Comment:
also subject for a change - no need to format it into a RAM buffer when the texts are in PROGMEM